tests/test_utils.sh

```diff
@@ -710,6 +710,31 @@ function test_is_function {
   unalias not_my_function
 }
 
+function test_hash_color {
+    PS1="$ "
+    lp_activate --no-config # For having _lp_foreground
+    _lp_hash_color "Debug"
+    assertContains "$lp_hash_color" "Debug"
+    # FIXME How to test color?
```

Comment:
Good question.

Since the tests can be run without color support (and in Github Actions will be), I think there is no valid test we can do to see if color was added. I think your test here is decent enough, as it will catch unset errors (like it already did).